addon/bootstrap.js

```diff
@@ -0,0 +1,118 @@
+/* globals AddonManager, Components, Services */
+
+const OLD_ADDON_PREF_NAME = "extensions.jid1-NeEaf3sAHdKHPA@jetpack.deviceIdInfo";
+const OLD_ADDON_ID = "jid1-NeEaf3sAHdKHPA@jetpack";
+const ADDON_ID = "screenshots@mozilla.org";
+const TELEMETRY_PREF = "toolkit.telemetry.enabled";
+const PREF_BRANCH = "extensions.screenshots.";
+const USER_DISABLE_PREF = "extensions.screenshots.disabled";
+const SYSTEM_DISABLE_PREF = "extensions.screenshots.system-disabled";
+
+const { interfaces: Ci, utils: Cu } = Components;
+Cu.import("resource://gre/modules/AddonManager.jsm");
+Cu.import("resource://gre/modules/Console.jsm");
+Cu.import("resource://gre/modules/Services.jsm");
+const { EmbeddedExtensionManager } = Cu.import("resource://gre/modules/LegacyExtensionsUtils.jsm");
```

Comment:
Can you just use the exposed LegacyExtensionsUtils instead since you don't need the untrack call below?